tests/bidi/playwright.config.ts

```diff
@@ -27,10 +27,14 @@ const trace = !!process.env.PWTEST_TRACE;
 const hasDebugOutput = process.env.DEBUG?.includes('pw:');
 
 function firefoxUserPrefs() {
+  const defaultPrefs = {
+    'network.proxy.allow_hijacking_localhost': true,
```

Comment:
I wonder if these settings make sense as Bidi defaults and should be added to the upstream file in Puppeteer that `packages/playwright-core/src/server/bidi/third_party/firefoxPrefs.ts` is derived from.